utils/build.ps1

```diff
@@ -1401,6 +1401,7 @@ function Build-Compilers() {
         SWIFT_ENABLE_EXPERIMENTAL_STRING_PROCESSING = "YES";
         SWIFT_ENABLE_SYNCHRONIZATION = "YES";
         SWIFT_ENABLE_VOLATILE = "YES";
+        SWIFT_SHOULD_BUILD_EMBEDDED_STDLIB_CROSS_COMPILING = "YES";
```

Comment:
This is the wrong place to do this. We do not build the runtimes as part of the compilers build.